Sources/LingoHubCLI/LingoHubCLI.swift

```diff
@@ -225,4 +234,44 @@ open class LingoHubCLI: NSObject, URLSessionDelegate, URLSessionDataDelegate {
         .resume()
     }
   }
+  
+  func stripPlaceholderLabels(for path: String) {
+    var sourceLocalizations: [String] = []
+    var cleanedLocalizations: [String] = []
+    do {
+      // Get the content
+      let contents = try NSString(contentsOfFile: path,
+                                  encoding: String.Encoding.utf8.rawValue)
+      sourceLocalizations = contents.components(separatedBy: "\n")
+    } catch let error as NSError {
+      print("Can't read file: \(error)")
+      exit(EXIT_FAILURE)
+    }
+    guard sourceLocalizations.count > 0 else {
+      return
+    }
+    var i = 0
+    while i < sourceLocalizations.count {
+      let localization = sourceLocalizations[i]
+      // In case of <> detection we ignore comment, localization itself
+      // and next empty line
+      if localization.range(of: #"<.+>"#, options: .regularExpression) != nil {
+        i += 3
```

Comment:
Jumping 3 lines works for perfectly formatted strings files, but can cause problems if someone misformats a file, uses <> in the base localization file, or similar... We need more robust logic here please...

Reply:
I changed the logic to ignore each string with placeholder separately and jump over next line if it's empty.